config/checkstyle/checkstyle.xml

```diff
@@ -149,7 +149,7 @@
         <module name="HideUtilityClassConstructor"/>
           <module name="OneTopLevelClass"/>
         <module name="MethodName">
-            <property name="format" value="^[a-z][a-zA-Z0-9_]*$"/>
+            <property name="format" value="^[a-zA-Z][a-zA-Z0-9_]*$"/>
```

Comment:
You should not change this. The format suggest camel case starting with lowercase. Why do we need it to start with upper case letter?